subsys/ll_sched/Kconfig

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig LL_SCHEDULER
+	bool "LL Scheduler"
```

Comment:
call it out here please, LL meaning "Low Level"

Reply:
Done.